src/react-treeview.jsx

```diff
@@ -38,9 +38,7 @@ const TreeView = React.createClass({
       <div
         {...rest}
         className={className + ' ' + arrowClassName}
-        onClick={this.handleClick}>
-        ▾
-      </div>;
```

Comment:
I'd keep the closing </div> as that is the regular HTML 4 format, the self-closing div is valid only in XHTML.